consensus/statehash.go

```diff
@@ -18,9 +18,9 @@ func (s *State) sortedUtxoSet() (sortedOutputs []Output) {
 	for _, utxoString := range unspentOutputStrings {
 		var outputID OutputID
 		copy(outputID[:], utxoString)
-		output, err := s.output(outputID)
-		if err != nil {
-			panic(err)
+		output, exists := s.output(outputID)
+		if !exists {
+			panic("output doesn't exist?")
```

Comment:
had to double check this one, but it looks like `sortedUtxoSet` is only ever called when `DEBUG` is set.